live-build/hooks/300-create-classic-diff.binary

```diff
@@ -12,7 +12,7 @@ PREFIX=binary/boot/filesystem.dir
 (cd $PREFIX
  mkdir -p var/lib/classic
  apt-get clean
- apt-get -y install --reinstall -d apt locales base-files libapt-inst2.0 libapt-pkg5.0 ubuntu-keyring
+ apt-get -y install --reinstall -d apt locales base-files libapt-inst2.0 libapt-pkg5.0 ubuntu-keyring lsb-release
```

Comment:
We cannot remove lsb-release, it was allowed under the default apparmor profile so this would constitute a backwards incompatible change. Something do to in core-18 I'm afraid.